service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseManagementOrder.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CaseManagementOrder {
+    private final String hearingDate;
+    private final UUID hearingDateId;
```

Comment:
so I would consider changing the name of this... it's not the hearingDateId its the Id of the element that contains hearing details right?

Reply:
changed as 'id'